Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -439,6 +439,16 @@
     return nullptr;
 }
 
+RefPtr<WebExtensionContext> WebExtensionController::extensionContext(const UniqueIdentifier& uniqueIdentifier) const
+{
+    for (Ref context : m_extensionContexts) {
+        if (context->isLoaded() && context->uniqueIdentifier() == uniqueIdentifier)
```

Comment:
If it is in `m_extensionContexts` it is going to be loaded. So you can drop `context->isLoaded() && `.